### PR TITLE
Update UG for v1.1

### PR DESCRIPTION
**Pull request by HugeNoob**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [x] Documentation Update

## Description

Add/Update UG documentation for 
1. `help` 
2. `deleteAllPerson`
3. `editTask`
4. `deleteTask`

## Related Tickets & Documents

-   Related Issue #38 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [x] No, and this is why: Check deployment [here](https://docs-ug-v1-1.coordimate.pages.dev/DraftUserGuide)

### CODE REVIEW

File: docs/DraftUserGuide.md [hunk 1]
```diff
@@ -112,8 +112,8 @@ Examples:
 
 Output:
 
-- `Opened help window`
-  ![help message](images/helpMessage.png)
+- `Open a window, leading you to our user guide.`
+  ![help message](images/output/help_success.png)
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:116]:
```suggestion
- `Open a window, leading you to our user guide.`

  ![help message](images/output/help_success.png)
```

File: docs/DraftUserGuide.md [hunk 2]
```diff
@@ -293,10 +293,12 @@ deleteAllPerson
 Examples:
 
 - `deleteAllPerson`
+  - Deletes all persons in your contact list.
 
 Output:
 
-![deleteAllPerson success](images/deleteAllPerson_success.png)
+- All persons in the contact list are deleted.
+  ![deleteAllPerson success](images/deleteAllPerson_success.png)
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:301]:
```suggestion
- All persons in the contact list are deleted.

  ![deleteAllPerson success](images/deleteAllPerson_success.png)
```

File: docs/DraftUserGuide.md [hunk 3]
```diff
@@ -350,6 +352,43 @@ Output:
...
+  - Edits the title of the 2nd task to be `Book room` and the note to be `By Friday`.
+- `editTask`
+  - Negative example as the index is not specified.
+
+Output:
+
+- Title of task 1 is edited to `Call Caterer`.
+  ![editTask_success](images/output/editTask_success1.png)
+- Title of task 2 is edited to `Book room` and note is edited to `By Friday`.
+  ![editTask_success](images/output/editTask_success2.png)
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:384]:
```suggestion
- Title of task 1 is edited to `Call Caterer`.

  ![editTask_success](images/output/editTask_success1.png)

- Title of task 2 is edited to `Book room` and note is edited to `By Friday`.

  ![editTask_success](images/output/editTask_success2.png)
```

File: docs/DraftUserGuide.md [hunk 4]
```diff
@@ -387,32 +426,35 @@ Errors:
...
+- `deleteTask 2`
+  - Deletes the 2nd task in the task list.
+- `findTask Call` followed by `deleteTask 1`
+  - Deletes the 1st task in the results of the `findTask` command.
 
 Output:
 
-![deleteTask_success](images/output/deleteTask_success.png)
+- Deletes task 2.
+  ![deleteTask_success](images/output/deleteTask_success1.png)
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:452]:
```suggestion
- Deletes task 2.

  ![deleteTask_success](images/output/deleteTask_success1.png)
```

File: docs/DraftUserGuide.md [hunk 5]
```diff
@@ -387,32 +426,35 @@ Errors:
...
 
 Errors:
 
-![deleteTask_error](images/error/deleteTask_error.png)
+- Index specified is not available in the task list.
+  ![deleteTask_error](images/error/deleteTask_error1.png)
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:457]:
```suggestion
- Index specified is not available in the task list.

  ![deleteTask_error](images/error/deleteTask_error1.png)
```

**Review by jiakai-17 — CHANGES_REQUESTED** [review]
Thanks for the changes, just a small nit about the layout. I think adding a newline between bullet points and images would make it look less "cramped".

Another possible nit would be the red underline in the UI mockups. Perhaps you could look into removing them if it is unintentional and not too troublesome for you?

Thanks!

**Comment by HugeNoob** [comment]
> Thanks for the changes, just a small nit about the layout. I think adding a newline between bullet points and images would make it look less "cramped".
> 
> Another possible nit would be the red underline in the UI mockups. Perhaps you could look into removing them if it is unintentional and not too troublesome for you?
> 
> Thanks!

Since the Ui mockups are temporary, I don't see a need for them to look perfect. Will our UG be graded before we replace them with actual images? I'm admittedly a bit lazy to change all of them again, wdyt?

**Review by jiakai-17 — APPROVED** [review]

**Comment by jiakai-17** [comment]
> > Thanks for the changes, just a small nit about the layout. I think adding a newline between bullet points and images would make it look less "cramped".
> > Another possible nit would be the red underline in the UI mockups. Perhaps you could look into removing them if it is unintentional and not too troublesome for you?
> > Thanks!
> 
> @jiakai-17 Since the Ui mockups are temporary, I don't see a need for them to look perfect. Will our UG be graded before we replace them with actual images? I'm admittedly a bit lazy to change all of them again, wdyt?

Yeah sure, we can leave it as it is for now, and update them when the respective features are implemented.

**Review by freshcabbage123 — APPROVED** [review]
LGTM

**Pull request merged**